node/test/util/read_repo_ast_util.js

```diff
@@ -1490,6 +1498,54 @@ describe("readRAST", function () {
         assert.deepEqual(actualMerge, merge);
     }));
 
+    it("cherry-pick", co.wrap(function *() {
+        // Start out with a base repo having two branches, "master", and "foo",
+        // foo having one commit on top of master.
+
+        const start = yield repoWithCommit();
+        const r = start.repo;
+
+        // Switch to master
+
+        yield r.checkoutBranch("master");
+
+        const head = yield r.getHeadCommit();
+        const sha = head.id().tostrS();
+
+        const cherryPick = new CherryPick(sha, sha);
```

Comment:
do you mean head, sha instead of sha, sha?

Reply:
No, it's writing out a cherry-pick that's from and to itself. More exhaustive testing of reading and writing the cherry-pick info is done in `CherryPickFileUtil`; this is just checking that the read APIs are in place.